resources/views/components/menu/account.blade.php

```diff
@@ -54,6 +54,7 @@
         <x-dropdown-item :link="route('user.completion-progress', $user)">Completion Progress</x-dropdown-item>
         
         @if($user->Permissions >= Permissions::Registered)
+            <x-dropdown-item :link="route('game.suggest')">Game Suggestions</x-dropdown-item>
             <x-dropdown-item :link="url('gameList.php?t=play')">Want to Play Games</x-dropdown-item>
```

Comment:
An autogenerated list seems like it should have slightly lower priority than the list of entities explicitly chosen by the user. Recommend keeping Want to Play Games above Game Suggestions.